### ISSUE TITLE
fine-tune on VQA

### ISSUE THREAD

**Issue by luomancs** [jackroos/VL-BERT#31]
Hi,

Thanks for your code. I want to use VLBERT to fine-tune on VQA dataset. The pretrained weights have 12-layer parameters. But in config file of VQA task, the number of layers is only 4. Can you provide more details about how to load pretrained weights and fine-tune on VQA?

Thank you.

**Comment by jackroos** [comment]
It is actually 12 layers for base model and 24 layers for large model. Please see the config yaml under the folder cfgs/vqa, e.g., https://github.com/jackroos/VL-BERT/blob/2ff7a65ac475b5e863e4dd5ab6792caeb7e1b1b4/cfgs/vqa/base_4x16G_fp32.yaml#L71.
For how to fine-tune the network for VQA, please see fine-tuning instructions in README. Thanks!

**Comment by jules-samaran** [comment]
Hi I'm also encountering an "issue" with fine-tuning on vqa, it seems that by default the model is trained on the train2014 + val2014 images and questions but val2014 is also used as the validation split. Why is that? Shouldn't the validation set bet holdout from training so that it gives independent information on which epoch model to choose for evaluation on test?

**Comment by jackroos** [comment]
@jules-samaran Following other's practice, we train on train+val split, and evaluate our model on test-dev and test-std by submitting to the VQA evaluation server. You can following the instructions on README for how to evaluate on test set. Thank you!
